### DISCUSSION TITLE
[Enhancement] Giving more love for WinUI sample app

### DISCUSSION THREAD

**Discussion by mhrastegari** [CommunityToolkit/Maui#1860] [New Feature Discussions]
With a little modification on Window and Shell handler I've achieved this:

Before:

![Screenshot (29)](https://github.com/CommunityToolkit/Maui/assets/42671084/c58dc209-0e17-4472-b3d7-78d554220611)

After:

![Screenshot (28)](https://github.com/CommunityToolkit/Maui/assets/42671084/94a90edd-f0ab-45c1-b129-c3a9a857cda9)

**Comment by TheCodeTraveler** [comment]
This looks great! Any chance you could submit a PR for this?

**Reply by mhrastegari** [reply]
With pleasure 🥰

**Reply by bijington** [reply]
It does look great! Is there a way some of the styling improvements could be applied to all platforms?

**Reply by mhrastegari** [reply]
Unfortunately as far as I know MacCatalyst does not support `BlendingMode.BehindWindow` :"( 
I've achieved that in Xamarin which uses Mac Cocoa but not MAUI.

And on other platforms like Android and iOS cannot be done since they're not desktop operating systems.

**Comment by CliffAgius** [comment]
Closed with PR #1861 , if closed in error please let us know to re-open.
